.github/workflows/linux.yml

```diff
@@ -450,10 +450,14 @@ jobs:
 
   free-threaded:
     needs: get_commit_message
+    strategy:
+      matrix:
+        parallel: ['0', '1']
     runs-on: ubuntu-latest
     if: >
       needs.get_commit_message.outputs.message == 1
       && (github.repository == 'scipy/scipy' || github.repository == '')
+      && false
```

Comment:
This disables the jobs completely. I think it's only needed to disable or remove the `parallel: 1` case? `parallel: 0` was already there and passes.